File: OPENAPI-REST-API/swagger-client/java-vertx/src/main/java/io/swagger/server/api/model/LoginSubmissionExample.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.server.api.model.LoginSubmissionExampleGrecaptcharesponse;
import io.swagger.v3.oas.annotations.media.Schema;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * The data to submit in the login request.
 */




public class LoginSubmissionExample   {
  private String login = null;

  private String passwd = null;

  private String remember = null;

  private LoginSubmissionExampleGrecaptcharesponse gRecaptchaResponse = null;

  private String tfa = null;

  public LoginSubmissionExample login(String login) {
    this.login = login;
    return this;
  }

  /**
   * Get login
   * @return login
   **/
    public String getLogin() {
    return login;
  }

  public void setLogin(String login) {
    this.login = login;
  }

  public LoginSubmissionExample passwd(String passwd) {
    this.passwd = passwd;
    return this;
  }

  /**
   * Get passwd
   * @return passwd
   **/
    public String getPasswd() {
    return passwd;
  }

  public void setPasswd(String passwd) {
    this.passwd = passwd;
  }

  public LoginSubmissionExample remember(String remember) {
    this.remember = remember;
    return this;
  }

  /**
   * Get remember
   * @return remember
   **/
    public String getRemember() {
    return remember;
  }

  public void setRemember(String remember) {
    this.remember = remember;
  }

  public LoginSubmissionExample gRecaptchaResponse(LoginSubmissionExampleGrecaptcharesponse gRecaptchaResponse) {
    this.gRecaptchaResponse = gRecaptchaResponse;
    return this;
  }

  /**
   * Get gRecaptchaResponse
   * @return gRecaptchaResponse
   **/
    public LoginSubmissionExampleGrecaptcharesponse getGRecaptchaResponse() {
    return gRecaptchaResponse;
  }

  public void setGRecaptchaResponse(LoginSubmissionExampleGrecaptcharesponse gRecaptchaResponse) {
    this.gRecaptchaResponse = gRecaptchaResponse;
  }

  public LoginSubmissionExample tfa(String tfa) {
    this.tfa = tfa;
    return this;
  }

  /**
   * Two Factor Authentication Response.
   * @return tfa
   **/
    public String getTfa() {
    return tfa;
  }

  public void setTfa(String tfa) {
    this.tfa = tfa;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    LoginSubmissionExample loginSubmissionExample = (LoginSubmissionExample) o;
    return Objects.equals(this.login, loginSubmissionExample.login) &&
        Objects.equals(this.passwd, loginSubmissionExample.passwd) &&
        Objects.equals(this.remember, loginSubmissionExample.remember) &&
        Objects.equals(this.gRecaptchaResponse, loginSubmissionExample.gRecaptchaResponse) &&
        Objects.equals(this.tfa, loginSubmissionExample.tfa);
  }

  @Override
  public int hashCode() {
    return Objects.hash(login, passwd, remember, gRecaptchaResponse, tfa);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class LoginSubmissionExample {\n");
    
    sb.append("    login: ").append(toIndentedString(login)).append("\n");
    sb.append("    passwd: ").append(toIndentedString(passwd)).append("\n");
    sb.append("    remember: ").append(toIndentedString(remember)).append("\n");
    sb.append("    gRecaptchaResponse: ").append(toIndentedString(gRecaptchaResponse)).append("\n");
    sb.append("    tfa: ").append(toIndentedString(tfa)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/login_submission_example.dart
part of swagger.api;

class LoginSubmissionExample {
  
  String login = null;

  String passwd = null;

  String remember = null;

  LoginSubmissionExampleGrecaptcharesponse gRecaptchaResponse = null;
/* Two Factor Authentication Response. */
  String tfa = null;

  LoginSubmissionExample();

  @override
  String toString() {
    return 'LoginSubmissionExample[login=$login, passwd=$passwd, remember=$remember, gRecaptchaResponse=$gRecaptchaResponse, tfa=$tfa, ]';
  }

  LoginSubmissionExample.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    login = json['login'];
    passwd = json['passwd'];
    remember = json['remember'];
    gRecaptchaResponse = new LoginSubmissionExampleGrecaptcharesponse.fromJson(json['g-recaptcha-response']);
    tfa = json['tfa'];
  }

  Map<String, dynamic> toJson() {
    return {
      'login': login,
      'passwd': passwd,
      'remember': remember,
      'g-recaptcha-response': gRecaptchaResponse,
      'tfa': tfa
     };
  }

  static List<LoginSubmissionExample> listFromJson(List<dynamic> json) {
    return json == null ? new List<LoginSubmissionExample>() : json.map((value) => new LoginSubmissionExample.fromJson(value)).toList();
  }

  static Map<String, LoginSubmissionExample> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, LoginSubmissionExample>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new LoginSubmissionExample.fromJson(value));
    }
    return map;
  }
}
